tools/vf-component-library/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 1.2.5
+
+* fix: Fix to replace common changelog to component specific changelog
```

Comment:
Since this was bug issue reported, can we update changelog accordingly.

Rename it as "Bug fix: Fixes identical changelog information in for all components #1874"

Reply:
Done  with latest commit.